app/scripts/models/reliers/oauth.js

```diff
@@ -31,7 +31,9 @@ define([
       // would have redirectUri as its base.
       redirectTo: null,
       // whether to fetch and derive relier-specific keys
-      keys: false
+      keys: false,
+      // always redirect to the rp, useful during email verification signup flow
+      alwaysRedirect: false
```

Comment:
I have to admit that I'm still a little confused by the name. Does `alwaysRedirect` mean the user is automatically redirected, or always given the option to redirect? If the second, could it be called `allowRedirect` or something like that? IIUC, [always_redirect](https://github.com/mozilla/fxa-content-server/pull/2366/files#diff-ca4d7096469c2a76694bb8214760cbaaR159) is specified by the relier?

Have other alternatives been considered, like making use of `redirectTo` or allowing the relier to specify an URL other than the redirectURI?

I'm trying to wrap my head around the relier HTTP API and am trying to avoid confusion (because I myself am confused).